resources/logging/values.yaml

```diff
@@ -203,8 +203,8 @@ global:
   logging:
     promtail:
       config:
-        name:
+        name: promtail-k8s-1-14.yaml
```

Comment:
> This value is valid if kyma deployed on kubernetes 1.14 or GKE 1.2.6.X [...]
We want to treat kubernetes >= 1.14 or GKE >= 1.2.6 as our **default** environment. It makes sense then to place values valid for such installation bundled with Kyma, i.e. inside the chart.
If we leave it empty as it is today we have a strange situation:
We force our users to make an additional step (create configmap with promtail) in order to make Kyma work in a default setup. At the same time we don't require such step for older or non-default setups.
In other words, we make the default installation **harder** than a non-default one. In addition we're punishing ourselves with repeating the same step again and again in our cluster-provisioning scripts. All of those with the same value: `promtail-k8s-1-14.yaml`

> we still have to support older kubernetes versions on GKE and other.
We do, and this PR does not change that. Users can always provide an explicit override with some other value (also an empty one) for `logging.promtail.config.name`. They must only know when such override is required. In my opinion, this should be documented in the logging component documents.

> We have to still document those version dependent setting with installation documentation [...]
That is exactly what we **don't want** to do. Installation docs should be focused on default, supported environment, and should be as short and simple as possible.
If we start to describe version-dependency differences of configuration values across Kyma components in the installation document, it will very quickly become incomprehensible.
Such nuances must be documented in respective component's documentation.
Perhaps we need a sort of compatibility-matrix in a central place, maybe even a new document in the **installation** category, but I think this is beyond scope of this PR.